src/design-tokens/layout.yml

```diff
@@ -0,0 +1,45 @@
+aliases:
+  width_30: '30em'
+  width_40: '40em'
+  width_48: '48em'
+  width_64: '64em'
+  width_80: '80em'
+  width_100: '100em'
+  width_120: '120em'
+props:
+  breakpoint_xs:
+    value: '{!width_30}'
+    type: media-query
+    category: media-query
+  breakpoint_sm:
```

Comment:
On a recent client project, we reduced some of these abbreviations down to a single letter. So instead of `xs`, `sm`, `md`, `lg`, `xl`, `xxl`, it was `xs`, `s`, `m`, `l`, `xl`, `xxl`.

I didn't really miss the extra letters. What do you think?